dronegen/buildbox.go

```diff
@@ -58,19 +58,21 @@ func buildboxPipelineStep(buildboxName string, fips bool) step {
 			`chown -R $UID:$GID /go`,
 			`docker login -u="$$QUAYIO_DOCKER_USERNAME" -p="$$QUAYIO_DOCKER_PASSWORD" quay.io`,
 			fmt.Sprintf(`make -C build.assets %s`, buildboxName),
-			fmt.Sprintf(`docker push quay.io/gravitational/teleport-%s:$RUNTIME`, buildboxName),
+			fmt.Sprintf(`docker push quay.io/gravitational/teleport-%s:$BUILDBOX_VERSION`, buildboxName),
 		},
 	}
 }
 
 func buildboxPipeline() pipeline {
 	p := newKubePipeline("build-buildboxes")
 	p.Environment = map[string]value{
-		"RUNTIME": goRuntime,
-		"UID":     {raw: "1000"},
-		"GID":     {raw: "1000"},
+		"BUILDBOX_VERSION": buildboxVersion,
+		"UID":              {raw: "1000"},
+		"GID":              {raw: "1000"},
 	}
-	p.Trigger = triggerPushMasterOnly
+
+	// only on master for now; add the release branch name when forking a new release series.
+	p.Trigger = pushTriggerForBranch("master")
```

Comment:
Would master and `branch/*` save us the need for changes later?

Reply:
Yeah, not a bad idea.